driver/test_kubeflow_workloads.py

```diff
@@ -144,6 +147,27 @@ async def test_create_profile(lightkube_client, create_profile):
 
     assert_namespace_active(lightkube_client, NAMESPACE)
 
+    # Sync of PodDefaults to the namespace can take up to 40 seconds
+    # Wait here is necessary to allow the creation of PodDefaults before Job is created
+    sleep_time_seconds = 40
```

Comment:
One thing came into my mind .... dont we want instead of fixed time for sleep have exponential retries with tenacity ? because for our CI in aks EKS we dont have to wait so weaiting 40 sec there is overkill. If we have the wait time with retries with tenacity we can start with some small sleep like few seconds. WDYT ?

Reply:
what should the retries check for tho? just the pipelines poddefault?

Reply:
what if we have the tenacity wait for pipelines poddefault + additional 10 seconds sleep for the others to be created?

Reply:
yeah waiting for pipelines poddefault sonds resonable to me and extra seconds for the rest

Reply:
we can skip the sleep if the filter is `not mlflow` i.e. the one used in our CI